xclim/indices/_agro.py

```diff
@@ -1163,12 +1157,14 @@ def standardized_precipitation_index(
 
     Notes
     -----
-    * The length `N` of the N-month SPI is determined by choosing the `window = N`.
+    * N-month SPI / N-day SPI is determined by choosing the `window = N` and the appropriate frequency `freq`.
     * Supported statistical distributions are: ["gamma", "fisk"], where "fisk" is scipy's implementation of
        a log-logistic distribution
     * If `params` is given as input, it overrides the `cal_start`, `cal_end`, `freq` and `window`, `dist` and `method` options.
+    * "APP" method only supports two-parameter distributions. Parameter `loc` needs to be fixed to use method `APP`.
     * The standardized index is bounded by ±8.21. 8.21 is the largest standardized index as constrained by the float64 precision in
       the inversion to the normal distribution.
+    * The results from `climate_indices` library can be reproduced with `method = "APP"` and `fitwkargs = {"floc": 0}`
```

Comment:
@aulemahal perhaps a notebook in the future could be warranted, but I think this is still helpful in the meantime

Reply:
Indeed. A "xclim compared to other package" page in the doc could be useful.

Reply:
Good idea. This also could be a good place to discuss the wrapper on SBCK, eventually Ibicus if we implement some things, etc ... :upside_down_face: